src/modules/todo/components/todo/Todo.tsx

```diff
@@ -1,34 +1,33 @@
-import type { TodoItem } from './Todo.interface'
 import { useForm } from 'react-hook-form'
-import { useFetchTodos } from '../../../../hooks/useFetchTodos'
 import * as classes from './Todo.module.scss'
+import { useMutationToPostTodo } from '../../hooks/useMutationToPostTodo'
+import { useFetchTodos } from '../../hooks/useFetchTodos'
```

Comment:
In general it would be good if you changed the path to `modules/todo/hooks`. Try to do it for all your imports.

Reply:
The solution I found has '@' at the beginning of the route

Reply:
Yes, that would work